src/AutoRest.CSharp/AutoRest/Plugins/CSharpGen.cs

```diff
@@ -73,10 +74,19 @@ public async Task<GeneratedCodeWorkspace> ExecuteAsync(Task<CodeModel> codeModel
                 project.AddGeneratedFile($"{responseHeaderModel.Type.Name}.cs", headerModelCodeWriter.ToString());
             }
 
+            if (configuration.PublicClients && !configuration.AzureArm && context.Library.Clients.Count() > 0)
```

Comment:
Do we have libraries without any clients?

Reply:
Yes, https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/mediaservices/Azure.Media.Analytics.Edge/src/Generated